app/scripts/controllers/network/createInfuraClient.test.js

```diff
@@ -1,389 +0,0 @@
-        await withMockedInfuraCommunications(async (comms) => {
-          const request = { method };
-
-          // The first time a block-cacheable request is made, the latest
-          // block number is retrieved through the block tracker first.
-          comms.mockNextBlockTrackerRequest({ blockNumber: '0x100' });
-          // This is our request.
-          comms.mockInfuraRpcCall({
-            request,
-            response: {
-              result: {
-                blockNumber: '0x200',
-              },
-            },
-          });
-          // The block-tracker-inspector middleware will request the latest
-          // block through the block tracker again.
-          comms.mockNextBlockTrackerRequest({ blockNumber: '0x300' });
-
-          await withInfuraClient(async ({ makeRpcCall, blockTracker }) => {
-            await makeRpcCall(request);
-            expect(blockTracker.getCurrentBlock()).toStrictEqual('0x300');
-          });
-        });
-      });
-    });
-
-    describe('eth_getTransactionCount', () => {
-      testsForRpcMethodSupportingBlockParam('eth_getTransactionCount', {
-        blockParamIndex: 1,
-      });
-    });
-
-    describe('eth_getTransactionReceipt', () => {
-      const method = 'eth_getTransactionReceipt';
-
-      testsForRpcMethodsThatCheckForBlockHashInResponse(method);
-
-      it("refreshes the block tracker's current block if it is less than the block number that comes back in the response", async () => {
-        await withMockedInfuraCommunications(async (comms) => {
-          const request = { method };
-
-          // The first time a block-cacheable request is made, the latest
-          // block number is retrieved through the block tracker first.
-          comms.mockNextBlockTrackerRequest({ blockNumber: '0x100' });
-          // This is our request.
-          comms.mockInfuraRpcCall({
-            request,
-            response: {
-              result: {
-                blockNumber: '0x200',
-              },
-            },
-          });
-          // The block-tracker-inspector middleware will request the latest
-          // block through the block tracker again.
-          comms.mockNextBlockTrackerRequest({ blockNumber: '0x300' });
-
-          await withInfuraClient(async ({ makeRpcCall, blockTracker }) => {
-            await makeRpcCall(request);
-            expect(blockTracker.getCurrentBlock()).toStrictEqual('0x300');
-          });
-        });
-      });
-    });
-
-    describe('eth_getUncleByBlockHashAndIndex', () => {
-      testsForRpcMethodAssumingNoBlockParam('eth_getUncleByBlockHashAndIndex');
-    });
-
-    describe('eth_getUncleByBlockNumberAndIndex', () => {
-      // NOTE: eth_getUncleByBlockNumberAndIndex does take a block param at the
-      // 0th index, but this is not handled by our cache middleware currently
```

Comment:
Nit: this comment was dropped

Reply:
It looks like this was moved to `shared-tests.js` (under "methods that assume there is no block param").

Reply:
That is a different comment. Notice that this method is not listed under that shared test.

Reply:
Fixed in[756e87d](https://github.com/MetaMask/metamask-extension/pull/16337/commits/756e87dd08aef9d8f75323faef496b8e0751e6ab).